src/formatter.rs

```diff
@@ -77,8 +79,10 @@ impl<'a> Formatter for RedisJsonFormatter<'a> {
         self.current_indent -= 1;
 
         if self.has_value {
-            writer.write_all(self.newline)?;
-            Self::indent(writer, self.current_indent, self.indent)?;
+            if let Some(n) = self.newline.as_ref() {
+                writer.write_all(n.as_bytes())?;
```

Comment:
Since repeated 4 times, can be considered to be moved to `RedisJsonFormatter` impl (to write an optional string)

Reply:
@oshadmi please check the last change